pipeline/test/test_pipeline.py

```diff
@@ -417,3 +417,24 @@ def test_verbose(self):
         t.command(f'cat {input}')
         p.write_output(input, f'{gcs_output_dir}/hello.txt')
         p.run(verbose=True)
+
+    def test_benchmark_lookalike_workflow(self):
+        p = self.pipeline()
+
+        setup_tasks = []
+        for i in range(10):
+            t = p.new_task(f'setup_{i}')
+            t.command(f'echo "foo" > {t.ofile}')
+            setup_tasks.append(t)
+
+        tasks = []
+        for i in range(1000):
+            t = p.new_task(f'create_file_{i}')
+            t.command(f'echo {setup_tasks[i % len(setup_tasks)].ofile} > {t.ofile}')
+            t.command(f'echo "bar" >> {t.ofile}')
```

Comment:
Make the CPU 0.1. That way we can pack them running 10 at a time on to the instance. But then this will take on the order of 100 seconds for this test alone adding the testing burden of all PRs by 2 minutes. Can we get away with 500 jobs and accomplish the same thing?

Reply:
Sure. For my education about batch, can you describe how you got to 100s?

Reply:
On my scale testing recently, it seemed like true jobs were taking about a second or so to run. I'm assuming echo is on the same order as that.

Reply:
I'm happier now that the ci test directory is getting cleaned up (I set the lifecycle to 1 day). Can you bump it and double check test_pipeline is still within 10 minutes or so?